### ISSUE TITLE
Creating our community forum for support and other questions

### ISSUE THREAD

**Issue by jurra** [osc-delft/osc-delft.github.io#27]
**Motivation**

**Proposal**

**Solution**

**Comment by emmyft** [comment]
Thanks @jurra for starting this!

🤗 What I like about this idea: I think it's kind of an ideal state to be in - where community members can help each other out, contribute to others work and ultimately build a culture of learning and sharing. I'd also like to further explore this with the community.

🤔 What I'm cautious about: In the current state, folks in research "support" roles (data stewards, RSEs, ICT, Open science programme staff) takes many of these questions from researchers - and rightfully so (that's our job!). I think managing the power dynamics in this community is 🔑  - you have paid staff in the community, and then researchers who are expected to volunteer their time to share and teach other people. We need to make sure that expectations are managed (e.g. no one is expected to complete others' research work), that contributions are correctly attributed and recognised, and staff roles have boundaries (e.g. there may be certain open solution that you'd like to advocate for, but you're there as a facilitator of discussions). OK those are basically my job to figure out (; but we need to make sure this is agreed.

Bottom line: 
- Adoption: the data champions Slack group is rather quiet - why would this fare any better? What do we uniquely offer? How do we promote this to people?
(note to self: I guess the entry point to this forum/community is _questions_, which is nice and effective for things that currently _don't_ have easy touchpoints - but say, I have a question about open access and I know Just is going to reply to my emails, why would I not just email Just?)
- Becoming a service-point for open science - completely fine (better have this than not!), but I guess not what we want ideally.

What is a prototype that we could test with? Maybe we could start with software...?

**Comment by jurra** [comment]
That is correct @emmyft 
We currently already support things, but we struggle to have a shared way of supporting the community, we have a shared inbox, slack channels, teams channels, but these are tools that dont comply with certain requirements:

**There are several interesting use cases for the forum:**
- When I write in the forum I send an email, also with specific context because of the tags and type of discussion
- We can capture discussions relevant for others
- People can search for questions that are basic before sending direct emails, so we reuse a lot of our knowledge
- Any one can answer questions or orient others, but also share experiences around topics
- We have a possibility to moderate discussions and facilitate them, this is also very useful

The forum is an enabler, but we need to use it and try it. My suggestion here is to first pilot it inside a smaller group of people, Datastewards, RSEs and DM as well as other members. They can be invited to the pilot.

Then after sometime exploring and learning we can decide if it is useful or not.

**Comment by jurra** [comment]
The prototype could be a simple [discourse droplet in digital ocean](https://marketplace.digitalocean.com/apps/discourse?utm_source=Customerio&utm_medium=Email_Internal&utm_campaign=Email_DiscourseWelcome&mkt_tok=eyJpIjoiWmpSalltRXlaakkyWmpaayIsInQiOiJobjZid0hjek9yUTQyTnBaTmVVVzVTSHR2NG04YnNTWDlyQXNjWExpMUlzR2VVNlJhaHp6d3R1TnZhdzFjTmtmVlN5QnduTDhGOTN1QkVwQzJVQ3RiNE9mK0FZWndCMmFndytXbFpOcUdpWGQzNnhTQkZVeXM0eVpwVWpcL0lvOVIifQ%3D%3D):

**Comment by emmyft** [comment]
Thanks @jurra -I' do think it's a good idea, and am definitely open to trying!

I just want to hone down on the hypotheses that we are testing with this prototype (I'm referring to community as a prototype here, beyond the infrastructure):
- People will search on this forum for existing knowledge before directly emailing support
- Community members will answer questions of other people
Feedback welcome! 🌟 

To test these, I think:
- Support (so RSEs, Data stewards and managers) needs to commit to replying openly on this forum and ONLY on this forum even if they had received the query via email - otherwise, if people continue to get direct support via email, they will not use the forum
- When community members ask questions on the forum, we put in the effort to loop in /mention community members that we think can help with the question, before answering it ourselves.

For Discourse - do you know if TU Delft is happy with this (in terms of the level of security?) I understood that we had some issues when we tried to move the data champions mailing list off the TUD listserv... As you mentioned, one of the biggest pros of discourse is to be able to email people with replies- may be something we need to figure out beforehand.

**Comment by jurra** [comment]
Agree @emmyft that is why is good to do a prelaunch, try it out internally first. 
Without a minimum commitment and alignment it will be useles because a forum without a community doesnt make any sense. 

With regard to discourse, you can host it anywhere you want, in digital ocean, linode, discourse, or TU delft servers. Is a service application. With regard to security, I guess digital ocean and linode are safer than TU Delft servers. Luckily it is also easier to setup with these providers. I already setup a droplet to test it but it needs to be configured, so maybe we can look into that together, and I can show you how that looks like in digital ocean.

**Comment by jurra** [comment]
By the way there is already a succesull community at TU Delft that is making use of discourse. It is led and maintained by Anton Akhmerove: https://forum.kavli.tudelft.nl/
He recommended using it as well, and confirmed that it was useful for his own purposes.
